runtime/src/staking.rs

```diff
@@ -701,6 +707,9 @@ decl_event!(
 		PermissionedValidatorRemoved(AccountId),
 		/// The given member was removed. See the transaction for who.
 		PermissionedValidatorStatusChanged(AccountId),
+        /// Remove the nominators from the valid nominators when there KYC expired
+        /// Caller, Stash accountId of nominators
+        InValidateNominator(AccountId, Vec<AccountId>),
```

Comment:
Very minor, should be "InvalidatedNominators" I think (other events seem to be generally past tense).